src/consensus.rs

```diff
@@ -52,14 +52,11 @@ pub use vote::Vote;
 use votor::Votor;
 
 /// Time bound assumed on network transmission delays during periods of synchrony.
-const DELTA: Duration = Duration::from_millis(400);
+const DELTA: Duration = Duration::from_millis(250);
 /// Time the leader has for producing and sending the block.
 const DELTA_BLOCK: Duration = Duration::from_millis(400);
-/// Timeout to use when we haven't seen any shred from the leader's block.
-/// This is used to skip honest but crashed leaders faster.
-const DELTA_EARLY_TIMEOUT: Duration = DELTA.checked_mul(2).unwrap();
-/// Timeout to use when we have seen at least one shred from the leader's block.
-const DELTA_TIMEOUT: Duration = DELTA_EARLY_TIMEOUT.checked_add(DELTA_BLOCK).unwrap();
+/// Base timeout for when leader's first slice should arrive if they sent it immediately.
+const DELTA_TIMEOUT: Duration = DELTA.checked_mul(3).unwrap();
```

Comment:
This is much better than what we had before.  

I would've expected to see a constant of something like "typical_time_to_create_first_slice" and `DELTA_TIMEOUT` defined as DELTA + this constant.

Another bit not being considered here is something about maximum clock drift between nodes.  Is that something that we should account for maybe?